### PR TITLE
fix load running time

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
fix pipeline initiailize load running time interval

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/pipeline/pipengine/engine.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func (engine *Engine) loadRunningPipelines() error {
 
 	// send pipeline id by interval time instead of at once
 	total := len(pipelineIDs)
-	intervalSec := time.Duration(conf.InitializeSendRunningIntervalSec())
```
Comment (sfwn) [inline modules/pipeline/pipengine/engine.go:112]: How long does it take for 600 workflows?

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/pipengine/engine.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/pipengine/engine.go:112]: 5 minutes if use default config

**Review by sfwn — APPROVED** [review]

**Pull request merged**